src/workerd/jsg/modules-new.c++

```diff
@@ -39,7 +53,26 @@ bool ensureInstantiated(Lock& js,
   return true;
 }
 
-ModuleBundle::Type toModuleBuilderType(ModuleBundle::BuiltinBuilder::Type type) {
+constexpr ResolveContext::Type moduleTypeToResolveContextType(Module::Type type) {
+  switch (type) {
+    case Module::Type::BUNDLE: {
+      return ResolveContext::Type::BUNDLE;
+    }
+    case Module::Type::BUILTIN: {
+      return ResolveContext::Type::BUILTIN;
+    }
+    case Module::Type::BUILTIN_ONLY: {
+      return ResolveContext::Type::BUILTIN_ONLY;
+      break;
```

Comment:
This line is not needed

Reply:
For stuff like this please just use a suggested edit ;-)